src/FSharp.MetadataFormat/FSharp.MetadataFormat.fsproj

```diff
@@ -119,7 +119,7 @@
     </ProjectReference>
   </ItemGroup>
   <Choose>
-    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And ($(TargetFrameworkVersion) == 'v4.0')">
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.0'">
```

Comment:
Yes we will evaluate that after we know what ms plans to change to MSBuild for dnc. It doesn't make too much sense right now